unit-tests/shared/src/test/scala/javalib/util/zip/CRC32Test.scala

```diff
@@ -70,4 +70,10 @@ class CRC32Test {
     )
   }
 
+  @Test def updateEmptyArray(): Unit = {
+    val crc = new CRC32()
+    crc.update(Array.emptyByteArray)
+    assertEquals(crc.getValue(), 0)
+  }
```

Comment:
@ekrich if you checkout 2523b8d6c846e454fb951bbd8ca109ed6fec99f0 which adds this test, you can run it in the JVM and see it passes while running in Native fails. This is how I discovered the problem (JVM/native implementations were not matching) and this is how I found the "proper" solution (make the implementations match behavior).

Reply:
Thank-you for the explanation.